### PR TITLE
chore: convert API tests to node test

### PR DESCRIPTION
**Pull request by jkaster**
Convert tests that call the Looker API server to use node test runner

Test files renamed from `*.spec.ts` to `*.apitest.ts`

Only files with `.apitest` use the node test runner

### CODE REVIEW

**Review by josephaxisa — APPROVED** [review]
lgtm

File: packages/sdk-node/src/nodeSession.apitest.ts [hunk 1]
```diff
@@ -25,15 +25,16 @@
  */
 
 import * as fs from 'fs';
-// import 'whatwg-fetch';
+import * as process from 'node:process';
+import { describe, it } from 'node:test';
+import { expect } from 'expect';
 import { ApiConfigMap, boolDefault, defaultTimeout } from '@looker/sdk-rtl';
-import { TestConfig } from '@looker/sdk-codegen-utils';
+import { TestConfig } from '@looker/sdk-rtl/src/testUtils';
```
Comment (bryans99) [inline packages/sdk-node/src/nodeSession.apitest.ts:32]: Do we need to reach into the package?

**Review by bryans99 — APPROVED** [review]
Questions and minor nits but LGTM

File: packages/sdk-node/src/nodeSession.apitest.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-node/src/nodeSession.apitest.ts:32]: yes, unfortunately

File: packages/sdk-node/test/sdk.apitest.ts [hunk 2]
```diff
@@ -423,11 +420,11 @@ describe('LookerNodeSDK integration tests', () => {
   });
 
   describe('User CRUD-it checks', () => {
-    beforeAll(async () => {
+    before(async () => {
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-node/test/sdk.apitest.ts:423]: no, that's the node test runner `beforeAll`

File: packages/sdk-node/test/sdk.apitest.ts [hunk 3]
```diff
@@ -608,7 +605,8 @@ describe('LookerNodeSDK integration tests', () => {
         let count = 0;
         let actual: IDashboard[] = [];
         const aggregate = (page: IDashboard[]) => {
-          console.log(`Page ${++count} has ${page.length} items`);
+          count++;
+          // console.log(`Page ${count} has ${page.length} items`);
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-node/test/sdk.apitest.ts:609]: leaving that in until I decide what to do with it. Should have marked it todo. Not a blocker.

File: packages/sdk-node/test/sdk.apitest.ts [hunk 4]
```diff
@@ -631,7 +629,8 @@ describe('LookerNodeSDK integration tests', () => {
...
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/sdk-node/test/sdk.apitest.ts:633]: in follow-up

**Pull request merged**